tests/secrets/cat.test.ts

```diff
@@ -61,6 +61,22 @@ describe('commandCatSecret', () => {
     expect(result.exitCode).toBe(0);
     expect(result.stdout).toBe(secretContent);
   });
+  test('should fail when reading root without secret path', async () => {
+    const vaultName = 'vault' as VaultName;
+    const vaultId = await polykeyAgent.vaultManager.createVault(vaultName);
+    const secretName = 'secret-name';
+    const secretContent = 'this is the contents of the secret';
+    await polykeyAgent.vaultManager.withVaults([vaultId], async (vault) => {
+      await vaultOps.addSecret(vault, secretName, secretContent);
+    });
+    const command = ['secrets', 'cat', '-np', dataDir, vaultName];
+    const result = await testUtils.pkStdio(command, {
+      env: { PK_PASSWORD: password },
+      cwd: dataDir,
+    });
+    expect(result.exitCode).not.toBe(0);
+    expect(result.stderr).toInclude('ErrorSecretsIsDirectory');
```

Comment:
The error formatting is currently not consistent, as we recently started using error objects. This has only been implemented for `mkdir` and not for other handlers. An issue already exists to track this, and this will be done in other PRs.

Reply:
The tracked issue is https://github.com/MatrixAI/Polykey/issues/826